packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx

```diff
@@ -94,29 +96,39 @@ export class Dialog extends BaseComponent<IDialogProps, IDialogState> {
         firstFocusableSelector={ firstFocusableSelector }
         forceFocusInsideTrap={ forceFocusInsideTrap }
         ignoreExternalFocusing={ ignoreExternalFocusing }
-        isBlocking={ isBlocking }
         isClickableOutsideFocusTrap={ isClickableOutsideFocusTrap }
-        isDarkOverlay={ isDarkOverlay }
-        isOpen={ hidden !== undefined ? !hidden : isOpen }
         onDismissed={ onDismissed }
         onLayerDidMount={ onLayerDidMount }
         responsiveMode={ responsiveMode }
         { ...modalProps }
-        className={ css('ms-Dialog', modalProps.className) }
-        containerClassName={ css(containerClassName, styles.main, modalProps.containerClassName) }
-        onDismiss={ modalProps.onDismiss || onDismiss }
-        subtitleAriaId={ this._getAriaLabelId(modalProps.subtitleAriaId ? modalProps.subtitleAriaId : ariaDescribedById, subText, id + '-subText') }
-        titleAriaId={ this._getAriaLabelId(modalProps.titleAriaId ? modalProps.titleAriaId : ariaLabelledById, title, id + '-title') }
+        isDarkOverlay={ isDarkOverlay ? isDarkOverlay : modalProps.isDarkOverlay }
```

Comment:
All of these should be testing whether the value is undefined, rather than simply for truthiness, since they are inherently booleans